docker/wait-for-dependencies.sh

```diff
@@ -1,11 +1,38 @@
-#!/bin/sh
+#!/bin/bash
+
  set -e
 
+until [ -f /var/lib/chroma/iml-settings.conf ]; do
+  echo "Waiting for settings."
+  sleep 1
+done
+
+TMP=$PROXY_HOST
+TMP2=$AMQP_BROKER_HOST
+TMP3=$DB_HOST
+
+set -a
+source /var/lib/chroma/iml-settings.conf
+set +a
+
+
+if [[ ! -z "$TMP" ]]; then
```

Comment:
So it's possible these variables didn't get set when it loaded the settings file?

Reply:
This is used by a bunch of upstream dockerfiles. Some of them will have the vars set, some will not.

Reply:
ok that makes sense.

Reply:
This whole bit is messy really, I need a way to extract all env vars nicely into docker-compose, but haven't found a DRY way to do that with both compose and swarm.